source/deployment/cluster.rst

```diff
@@ -339,6 +339,13 @@ If ``"DriverName": "local"`` is used then the directory at ``"FileSettings":`` `
 
 Note a slight behavior change in 5.15: When you reinstall a plugin in 5.14, the previous **Enabled** or **Disabled** state is retained. As of 5.15, a reinstalled plugin's initial state is **Disabled**.
 
+CLI and High Availability
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The CLI is run in a single node which bypasses the mechanisms that a `High Availability environment <https://docs.mattermost.com/deployment/cluster.html>`__ uses to perform actions across all nodes in the cluster. As a result, when running `CLI commands <https://docs.mattermost.com/administration/command-line-tools.html>`__ in a High Availability environment, tasks such as creating and deleting users or changing configuration settings require a server restart.
```

Comment:
Creating users should not require a server restart, so I wouldn't use it as an example. I'd suggest to use use updating instead:

```suggestion
The CLI is run in a single node which bypasses the mechanisms that a `High Availability environment <https://docs.mattermost.com/deployment/cluster.html>`__ uses to perform actions across all nodes in the cluster. As a result, when running `CLI commands <https://docs.mattermost.com/administration/command-line-tools.html>`__ in a High Availability environment, tasks such as updating and deleting users or changing configuration settings require a server restart.
```